.travis.yml

```diff
@@ -47,7 +47,17 @@ matrix:
     # Linux builds
     #
     # GCC 9
-    - env: COMPILER=g++-9 BUILD_SINGLE_HEADER=true
+    - language: python
+      python:
+        - "3.7"
+      dist: xenial
+      install:
+        - pip install conan==1.10.2 conan-package-tools
```

Comment:
Minor one, I understand the way to install is:
```bash
pip install conan-package-tools
```
As then conan-package-tools will automatically fetch the correct version of Conan for self anyway.  Otherwise, version miss-match issues can occur between conan and conan-package-tools.